app/models/concerns/products/schedule_rule_support.rb

```diff
@@ -9,6 +9,8 @@ module Products::ScheduleRuleSupport
     has_many :product_access_groups, foreign_key: :product_id, inverse_of: :product
   end
 
+  PRICING_MODES = ["Schedule Rule", "Duration"].freeze
```

Comment:
Since we only want this on instruments, can you move this back to the `Instrument` class?